src/translations.js

```diff
@@ -94,75 +94,74 @@ module.exports = class Translation {
   spawnTranslations () {
     let self = this;
     let alpha = moment();
-    if(process.env.ORGANISATION=="CH"){
-      var options = {
-        url: this.config._LOKALISE_BASE_URL+this.config._CEMBRA_PROJECT_ID+"/files/download",
-        headers: {
-            "x-api-token": this.config._LOKALISE_API_TOKEN
-        },
-        json: {
-            "format": "json",
-            "original_filenames": false
-        },
-        gzip: true
-      }
-      request.post(options, function (err, data) {
-          if (err) {
-              return err;
+      console.info(`Spawning Translations started for ${self.config._LOCALES}`);
+      Promise.map(self.config._LOCALES, (translationLocale) => {
+        if(translationLocale.toLowerCase()=="ch"){
```

Comment:
is would be nice to have this separated out in a function, just for better code readability. Other wise implementation is ok.